SpatialGDK/Source/SpatialGDKEditor/Private/SpatialLaunchConfigCustomization.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialLaunchConfigCustomization.h"
+
+#include "SpatialGDKSettings.h"
+#include "SpatialGDKEditorSettings.h"
+
+#include "PropertyCustomizationHelpers.h"
+#include "PropertyHandle.h"
+#include "Widgets/SToolTip.h"
+#include "IDetailChildrenBuilder.h"
+#include "Widgets/Text/STextBlock.h"
+#include "IDetailGroup.h"
+
+TSharedRef<IPropertyTypeCustomization> FSpatialLaunchConfigCustomization::MakeInstance()
+{
+	return MakeShared<FSpatialLaunchConfigCustomization>();
+}
+
+void FSpatialLaunchConfigCustomization::CustomizeHeader(TSharedRef<class IPropertyHandle> StructPropertyHandle, class FDetailWidgetRow& HeaderRow, IPropertyTypeCustomizationUtils& StructCustomizationUtils)
+{
+	
+}
+
+void FSpatialLaunchConfigCustomization::CustomizeChildren(TSharedRef<class IPropertyHandle> StructPropertyHandle, class IDetailChildrenBuilder& StructBuilder, IPropertyTypeCustomizationUtils& StructCustomizationUtils)
+{
+	TArray<UObject*> EditedObject;
+	StructPropertyHandle->GetOuterObjects(EditedObject);
+
+	if (EditedObject.Num() == 0)
+	{
+		return;
+	}
+
+	bool bIsInSettings = Cast<USpatialGDKEditorSettings>(EditedObject[0]) != nullptr;
+
+	uint32 NumChildren;
+	StructPropertyHandle->GetNumChildren(NumChildren);
+	for (uint32 ChildIdx = 0; ChildIdx < NumChildren; ++ChildIdx)
+	{
+		USpatialGDKEditorSettings* EditorSettings = Cast<USpatialGDKEditorSettings>(EditedObject[0]);
```

Comment:
can we do the cast once before the loop for both EditorSettings and bIsInSettings?